atlasdb-client/src/main/java/com/palantir/atlasdb/cache/OffHeapTimestampCache.java

```diff
@@ -126,6 +149,7 @@ private static CacheDescriptor createNamespaceAndConstructCacheProposal(
         @Override
         public Map<Map.Entry<Long, Long>, Map.Entry<Long, Long>> apply(Set<Map.Entry<Long, Long>> request) {
             if (offHeapTimestampCache.cacheDescriptor.get().currentSize().get() >= offHeapTimestampCache.maxSize) {
+                offHeapTimestampCache.taggedMetricRegistry.meter(CACHE_NUKE).mark();
```

Comment:
Hmm. A meter is plausible here, but it registers a bunch of series that I'm not sure are interesting for an operation that I don't think is super common.

Reply:
Ack.